lib/modules/marker.rb

```diff
@@ -0,0 +1,29 @@
+module Codebreaker
+  module Marker
+    def check_win
+      if win?
+        'Congratulation! You win!'
+      elsif @attempts.zero?
+        @hint = 0
+        @game_start = false
+        "Game over! Secret code is #{@secret_code}."
+      else
+        @attempts -= 1
+        '+' * pluses + '-' * (minuses - pluses)
+      end
+    end
+
+    def pluses
+      @player_code.to_s.chars.to_a.map.with_index { |num, index| '+' if num == @secret_code[index] }.compact.size
```

Comment:
try to stick to max 120 chars per line convention. dat is really hard to read such long lines